docs/api/parameters.md

```diff
@@ -0,0 +1,208 @@
+# Parameters
+
+> This page is incomplete, stay tuned.
+
+## Get parameters
+
+### Code samples
+
+```shell
+# Example request using curl
+curl -X GET http://coder-server:8080/api/v2/parameters/{scope}/{id} \
+  -H 'Accept: application/json' \
+  -H 'Coder-Session-Token: API_KEY'
+```
+
+`GET /parameters/{scope}/{id}`
+
+### Parameters
+
+| Name    | In   | Type         | Required | Description |
+| ------- | ---- | ------------ | -------- | ----------- |
+| `scope` | path | string       | true     | Scope       |
+| `id`    | path | string(uuid) | true     | ID          |
+
+#### Enumerated Values
+
+| Parameter | Value        |
+| --------- | ------------ |
+| `scope`   | `template`   |
+| `scope`   | `workspace`  |
+| `scope`   | `import_job` |
+
+### Example responses
+
+> 200 Response
+
+```json
+[
+  {
+    "created_at": "string",
+    "destination_scheme": "none",
+    "id": "497f6eca-6276-4993-bfeb-53cbbbba6f08",
+    "name": "string",
+    "scope": "template",
+    "scope_id": "5d3fe357-12dd-4f62-b004-6d1fb3b8454f",
+    "source_scheme": "none",
+    "updated_at": "string"
+  }
+]
+```
+
+### Responses
+
+| Status | Meaning                                                 | Description | Schema                                                      |
+| ------ | ------------------------------------------------------- | ----------- | ----------------------------------------------------------- |
+| 200    | [OK](https://tools.ietf.org/html/rfc7231#section-6.3.1) | OK          | array of [codersdk.Parameter](schemas.md#codersdkparameter) |
+
+<h3 id="get-parameters-responseschema">Response Schema</h3>
+
+Status Code **200**
+
+| Name                   | Type   | Required | Restrictions | Description                                       |
+| ---------------------- | ------ | -------- | ------------ | ------------------------------------------------- |
+| `[array item]`         | array  | false    |              | [Parameter represents a set value for the scope.] |
+| `» created_at`         | string | false    |              |                                                   |
+| `» destination_scheme` | string | false    |              |                                                   |
+| `» id`                 | string | false    |              |                                                   |
+| `» name`               | string | false    |              |                                                   |
+| `» scope`              | string | false    |              |                                                   |
+| `» scope_id`           | string | false    |              |                                                   |
+| `» source_scheme`      | string | false    |              |                                                   |
+| `» updated_at`         | string | false    |              |                                                   |
```

Comment:
Both updated and created at are datetimes, right? Should we mark them as such?

Reply:
Fixed